game.js

```diff
@@ -9,46 +9,29 @@ Game.prototype.play = function(){
 	debugger;
 	currentPlayer=this.currentPlayer;
 	if(currentPlayer===0){
-		randomOrder[0].placeMarker();
+		randomOrder[0].placeMarker(this);
```

Comment:
The player object is going to need to know the state of the game object, so you could pass the Game object to the player. Alternatively, you can do two-way binding, but that seems messy to me.